### ISSUE TITLE
Incorrect error message with missing whitespace in MultiPCA

### ISSUE THREAD

**Issue by bcipolli** [nilearn/nilearn#444]
Current code and error message:
```python
                 if self.n_components > subject_pca.shape[0]:
                    raise ValueError('You asked for %i components.'
                                     'This is smaller than single-subject '
                                     'data size.' % self.n_components)
```

Two fixes, one suggestion:
* `smaller` should be `larger`.
* Whitespace is missing between the two sentences.
* Suggest: also output the single-subject data size, to help the user debug.

I coded this on my machine as:
```python
                 if self.n_components > subject_pca.shape[0]:
                    raise ValueError('You asked for %i components. '
                                     'This is larger than the single-subject '
                                     'data size (%d).' % (self.n_components,
                                                          subject_pca.shape[0]))
```

**Comment by lesteve** [comment]
Good catch and I agree on all your points! For small fixes like this don't hesitate to do a PR directly, it does reduce the overhead a little bit.

Bonus message if you add a test where you check the error message if there is none already.

**Comment by bcipolli** [comment]
Cool--glad to do a PR directly for these types of small changes.
I will take a look at the test situation and do a PR sometime soon!

**Comment by AlexandreAbraham** [comment]
:+1: 

> For small fixes like this don't hesitate to do a PR directly

Or just push it into master :P (private joke for @lesteve).

**Comment by lesteve** [comment]
> Or just push it into master :P (private joke for @lesteve).

:laughing:

**Comment by bcipolli** [comment]
lol :)  Fixed in #447.
